sources/packages/backend/apps/api/src/route-controllers/application-exception/_tests_/e2e/application-exception.aest.getPendingApplicationExceptions.e2e-spec.ts

```diff
@@ -25,19 +25,23 @@ describe("ApplicationExceptionAESTController(e2e)-getPendingApplicationException
     // Arrange
     const application1Promise = saveFakeApplicationWithApplicationException(
       appDataSource,
-      ApplicationExceptionStatus.Pending,
+      undefined,
+      { applicationExceptionStatus: ApplicationExceptionStatus.Pending },
```

Comment:
The factory `saveFakeApplicationWithApplicationException` is expected to create the exceptions in Pending status and by ensuring that, we can avoid explicitly using the exception status here.

Reply:
https://github.com/bcgov/SIMS/pull/1952#discussion_r1207315678

Reply:
With changes in saveFakeApplicationWithApplicationException we can remove line 28,29,33,34 now.